test/policies/portfolio_policy_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require "test_helper"
```

Comment:
It looks like we still have some issues with the linter 🤔 

If you run `bundle exec rubocop -A` or `bin/dc rubocop -A`, it will be fixed by rubocop. 

<img width="800" height="231" alt="image" src="https://github.com/user-attachments/assets/34abefec-c58c-4140-b393-9cfb3b727c86" />